### DISCUSSION TITLE
Alerting notification policies `group_by: status`

### DISCUSSION THREAD

**Discussion by Dima-Diachenko** [grafana/grafana#62977] [Alerting]
Hello Grafana community! 

Today I got complaints from our developers about Slack notifications and obvious easy to realize point how to improve it.
So we use new Grafana Alerting (on prem deployment), configured provisioning of the config and following notification policies:
```
  policies:
    - receiver: grafana-dev
      group_by: ['alertname', 'grafana_folder']
      group_wait: 5m
      group_internval: 5m
      repeat_interval: 24h
```
UI view:
![image](https://user-images.githubusercontent.com/66964765/217055019-bdb7eee9-b663-446e-9c55-75fcde4a7ba8.png)

With this setup we often catch alerts like this:
<img width="600" alt="image" src="https://user-images.githubusercontent.com/66964765/217049726-144a2484-8c3a-4954-8b8a-11176e7f1989.png">

Even if we have 99 RESOLVED alerts and 1 FIRING - we will see "red" notification message and it is counter-intuitive to understand that in fact this message means that 99/100 alerts were resolved.

I (and probably everybody) would be glad to have the option to separate alerts resolved from triggered. Just send then in 2 independent messages.

I think the solution of this is grouping by "status" of the alerts.
Frankly said, I already tried following configs:
 ```
 group_by: ['alertname', 'status']
```
or
```
group_by: ['alertname', 'alertstate']
```
But it doesn't work as expected.

I'm not alone with my pain, I found [this issue on GH](https://github.com/prometheus/alertmanager/issues/2334), but it's more about Prometheus Alertmanager. 
And we have new awesome Grafana unified Alerting! What would you recommend with this issue?

**Comment by Dima-Diachenko** [comment]
Hello, 

Could you take a look on it guys?

**Comment by gillesdemey** [comment]
This is currently a limitation of the AlertManager (which the new Alerting uses internally) since it only supports grouping by `label` and the status of the alert isn't a label. 

I recommend continuing the conversation in the issue you've linked and try to advocate for your use-case, there's currently no known work-around in Grafana.

**Reply by ghost** [reply]
Got you, thanks for the answer.
